gpjax/likelihoods.py

```diff
@@ -199,6 +199,42 @@ def predict(self, dist: tfd.Distribution) -> tfd.Distribution:
         return self.link_function(mean / jnp.sqrt(1.0 + variance))
 
 
+@dataclass
+class Beta(AbstractLikelihood):
+    def link_function(self, alpha: float, beta: float) -> tfd.Distribution:
+        r"""The probit link function of the Beta likelihood.
+
+        Args:
+            alpha : alpha parameter, indicating mean number of successes
+            beta  : beta parameter, indicating mean number of failures
+
+        Returns
+        -------
+            tfd.Distribution: The likelihood function.
+        """
+        return tfd.Beta(alpha, beta)
```

Comment:
I think with the above change I've suggested, you'll need to call the inverse probit function in here, similar to the Bernoulli likelihood.

Reply:
Should the inverse probits be applied something like this:

`return tfd.Beta(concentration1=inv_probit(alpha), concentration1=inv_probit(beta))` ?

I'm not entirely sure what this does, is it just to normalize the alpha and beta parameters to [0, 1]?